simpleheat.js

```diff
@@ -128,5 +128,15 @@ simpleheat.prototype = {
                 pixels[i + 2] = gradient[j + 2];
             }
         }
+    },
+    
+    _createCanvas:function() {
+        if (typeof document !== 'undefined') {
+            return document.createElement('canvas');
+        } else {
+            //works for node-canvas, but others?
+            //any better implementations?
+            return new this._canvas.constructor(1, 1);
```

Comment:
in node-canvas, canvas instances are usually created by:

```javascript
var canvas = new Canvas(width, height)
```
so 1, 1 is the initial width and height. I removed it as node-canvas also has a default constructor without any parameter.